src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -42,11 +42,25 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 	}
 
 	dotnetBinary := "/usr/share/dotnet/dotnet"
+
 	enableDynamicInstall := common.GetBooleanEnvironmentVariable(consts.EnableDynamicInstallKey)
 	if enableDynamicInstall && !common.PathExists(dotnetBinary) {
 		scriptBuilder.WriteString(fmt.Sprintf("oryx setupEnv -appPath %s\n", appPath))
 	}
 
+	preRunCommandOrScript := common.GetEnvironmentVariable(consts.PreRunCommandOrScriptEnvVarName)
+	if preRunCommandOrScript != "" && !common.PathExists(dotnetBinary) {
+		if common.FileExists(preRunCommandOrScript) {
```

Comment:
Possible values for `preRunCommandOrScript`: 
- `,/installPreqs.sh`
- `apt-get install foo`
- `./installPreqs.sh && apt-get install foo`

To make it less complicated for ourselves, we can avoid parsing the content and checking if a file exists or not. We can tell users that the runtime script is always run from root of the app directory. Let's make sure that we run from the app directory.